resources/jolokia-access.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The latter probably applies here. The name for that file that I've seen most projects use is `jolokia-access.xml`.

We could definitely change it though, since we have control over the defaults set in `jolokia.clj`.